doc/rbd/rbd-openstack.rst

```diff
@@ -232,15 +233,28 @@ Edit ``/etc/glance/glance-api.conf`` and add under the ``[glance_store]`` sectio
 
 For more information about the configuration options available in Glance please refer to the OpenStack Configuration Reference: http://docs.openstack.org/.
 
-Any OpenStack version
-~~~~~~~~~~~~~~~~~~~~~
+Enable copy-on-write cloning of images
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Note that this exposes the back end location via Glance's API, so the endpoint
+with this option enabled should not be publicly accessible.
+
+Before Mitaka
+^^^^^^^^^^^^^
 
 If you want to enable copy-on-write cloning of images, also add under the ``[DEFAULT]`` section::
 
     show_image_direct_url = True
 
-Note that this exposes the back end location via Glance's API, so the endpoint
-with this option enabled should not be publicly accessible.
+As of Mitaka
+^^^^^^^^^^^^
+
+To enable image locations and take advantage of copy-on-write cloning for images, add under the ``[DEFAULT]`` section::
+
+    show_multiple_locations = True
+
+Disable cache management (any OpenStack version)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Minor: tested the docs and noticed this was not the same font size -- the "^"s should be replaced with "~"s 

http://docs.ceph.com/docs/wip-jd-testing/rbd/rbd-openstack/#disable-cache-management-any-openstack-version